encoders/nlp/TransformerTFEncoder/tests/test_transformertfencoder.py

```diff
@@ -28,27 +28,23 @@ def test_metas(tmpdir, random_workspace_name):
 
 encoders_parameters = [
     {
-        "pretrained_model_name_or_path": 'sentence-transformers/distilbert-base-nli-stsb-mean-tokens',
-        "model_save_path": 'distilbert-base-nli-stsb-mean-tokens',
-    },
-    {
-        "pretrained_model_name_or_path": 'bert-base-uncased',
-        "model_save_path": 'bert-base-uncased',
+        "pretrained_model_name_or_path": 'distilbert-base-uncased',
```

Comment:
would it be good to maybe test more models that are popular? and maybe build a matrix of parameters by using `pytest parametrize` to have kind of a cartesian product of different parameters for extensive testing? Like cover 3 or 4 popular models with each of them will different pooling strategy and even layer index?